package.json

```diff
@@ -66,7 +67,6 @@
     "@types/react-redux": "^7.1.7",
```

Comment:
This should be removed since react-redux 8 is written in Typescript and includes type information.

https://react-redux.js.org/using-react-redux/usage-with-typescript

Reply:
Exactly. It was a draft as I verified the removal of the types, as per the recommendations.